front/app/containers/Admin/pagesAndMenu/containers/GenericHeroBannerForm/BannerImageFields.tsx

```diff
@@ -225,16 +225,18 @@ const BannerImageField = ({
             </Warning>
           </Box>
         ) : (
-          <HeaderImageDropzone
-            onAdd={bannerImageAddHandler}
-            onRemove={bannerImageRemoveHandler}
-            overlayColor={bannerOverlayColor}
-            overlayOpacity={bannerOverlayOpacity}
-            headerError={bannerError}
-            header_bg={headerLocalDisplayImage}
-            previewDevice={previewDevice}
-            layout={bannerLayout || 'full_width_banner_layout'}
-          />
+          <Box data-cy="e2e-homepage-banner-image-dropzone">
```

Comment:
Do you need a Box to add data-cy?

Reply:
The test cannot find the element with this code:

```tsx
<HeaderImageDropzone
  data-cy="e2e-homepage-banner-image-dropzone"
...
```